packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -27,7 +27,7 @@ enableMapSet();
  */
 export class ChatContext {
   private _state: ChatClientState = {
-    userId: <UnknownIdentifierKind>{ id: '' },
```

Comment:
One of the case, Babel can't understand - <> cast, however, foo as Bar is always the recommended way.